src/debug_menu.cpp

```diff
@@ -126,14 +127,15 @@ void npc_edit_menu()
         for( const auto &need : np->needs ) {
             data << need << std::endl;
         }
+        data << string_format( _( "Total morale: %d" ), int( np->get_morale_level() ) ) << std::endl;
```

Comment:
`player::get_morale_level` already returns an `int`, why the cast?

Reply:
Thanks. That was copy-paste error.